### PR TITLE
Fix the CI build

### PR DESCRIPTION
**Pull request by jmg-duarte**
- Removes the ARM v6 architecture
- Removes OpenSSL as a dependency and where really needed, replaced it with `rustls-tls`
- Replaces a, so far, uncompiled path from the `ax` binary with a `panic`
- Adapts the Wix, Makefile and friends to build properly

Finally, artifacts (mainly Windows) still need testing

### CODE REVIEW

File: rust/actyx/ax/src/main.rs [hunk 1]
```diff
@@ -144,3 +149,75 @@ async fn with_logger<T>(fut: impl Future<Output = T>, verbosity: u8) -> T {
...
+        None => false,
+    };
+    let log_as_json = match log_json {
+        Some(Color::On) => true,
+        Some(Color::Off) => false,
+        Some(Color::Auto) => is_no_tty,
+        None => false,
+    };
+
+    // NOTE: maybe it's time we remove this this
```
Comment (rkuhn) [inline rust/actyx/ax/src/main.rs:180]: yes, please

File: wix/actyx-installer/Dockerfile [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# This image uses WiX 3.8
+# Check the following link for more information:
+# https://hub.docker.com/r/justmoon/wix/dockerfile
 FROM justmoon/wix
```
Comment (rkuhn) [inline wix/actyx-installer/Dockerfile:4]: I’d feel better if we depended on the specific hash

**Review by rkuhn — APPROVED** [review]
very nice!

File: wix/actyx-installer/Dockerfile (hunk 2 above)
Comment (jmg-duarte) [reply wix/actyx-installer/Dockerfile:4]: The image hasn't been updated in something like 8 or 4 years. But I get you

**Pull request merged**